include/swift/SIL/SILBridging.h

```diff
@@ -994,6 +999,22 @@ struct OptionalBridgedVTable {
   swift::SILVTable * _Nullable table;
 };
 
+struct BridgedConstExprFunctionState {
+  swift::ConstExprFunctionState * _Nonnull state;
+  swift::SymbolicValueBumpAllocator * _Nonnull allocator;
+  swift::ConstExprEvaluator * _Nonnull constantEvaluator;
+  unsigned int * _Nonnull numEvaluatedSILInstructions;
+  
+  SWIFT_IMPORT_UNSAFE BRIDGED_INLINE
+  static BridgedConstExprFunctionState create();
+  
+  SWIFT_IMPORT_UNSAFE BRIDGED_INLINE
```

Comment:
You don't need `SWIFT_IMPORT_UNSAFE` for functions which return a "trivial" value like `bool` or `void`. It's only need for functions which return some kind of object via a pointer.